packaging/docker/sidecar.py

```diff
@@ -226,7 +234,7 @@ def __init__(self):
             self.substitutions["FDB_MACHINE_ID"] = os.getenv("HOSTNAME", "")
 
         if self.substitutions["FDB_ZONE_ID"] == "":
-            self.substitutions["FDB_ZONE_ID"] = self.substitutions["FDB_MACHINE_ID"]
+            self.substitutions["FDB_ZONE_ID"] = self.read_node_zone_topology_label(self.substitutions["FDB_MACHINE_ID"]) if self.read_zone_topology_labels else self.substitutions["FDB_MACHINE_ID"]
```

Comment:
Should we move the if statement into the method? I believe that would improve the readability.